package com.sda.advanced.exceptions.error_vs_exception;

public class DemoError {

    public static void main(String[] args) {
        StackOverflow.work(5);
    }
}
